### ISSUE TITLE
DIFI:  1 Introduction and 3.5 Limitations, Assumptions or Exceptions

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act#201]
It is crucial to document the interpretation of the accessibility requirement/WCAG. Limitations, assumptions or exceptions at the accessibility requirement/WCAG level form basis for any limitations, assumptions or exceptions at test rule level.

Test rules may cover only parts of the requirement interpretation. Will this be documented in the section Assumption for each test rule? There should be an overview of which test rules cover which requirements, in order to be able to fully test compliance with each requirement.

**Comment by WilcoFiers** [comment]
The taskforce discussed the question and agreed that with each rule, it is already required to indicate what success criteria that rule generates a failure to. This gives an understanding of what is covered where. What ACT rules do not do is indicate when a web page passed a success criteria. While we acknowledge that occasionally it might be possible to infer conformance, the goal of ACT rules is to prove non-conformance.
